code/ttt-reborn.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This file is not from IDE, but from s&box project generation file.

Reply:
I don't think we need that file as well as we already have a `tttreborn.csproj` file. Not matching the repo's name, but the gamemode's name. If you'd change the add-on's folder into `tttreborn` as well, the generation tool would name this file like mine. But we can use your file as well, it doesn't matter, just don't need to have two of the same files :D

Reply:
Ah, didn't realize it was based on folder name. Assumed it was based off of `.addon` or something similar.

Reply:
Alright, then we have to remove `tttreborn.csproj` here as well :D